posts/2023-09-13-Primeur.adoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Open Liberty in Action - Primeur’s data integration platform"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/GraceJansen
+author_github: https://github.com/GraceJansen
+seo-title: Open Liberty in Action - Primeur’s data integration platform - OpenLiberty.io
+seo-description: Learn how Open Liberty proved to be the perfect pillar for Primeur, providing invisibility, observability and flexibility.
+blog_description: "Learn how Open Liberty proved to be the perfect pillar for Primeur, providing invisibility, observability and flexibility."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Open Liberty in Action - Primeur’s data integration platform
+Grace Jansen <https://github.com/GraceJansen>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+link:https://www.primeur.com/en/[Primeur], an independent software vendor (ISV) based in Italy, focuses on providing smart data integration. They embed Open Liberty Java runtime in their flagship data integration platform, DATA ONE. Open Liberty enables Primeur to make the embedded Java runtime invisible to their data integration customers, while still observable to Primeur’s engineers to be able to support them.
+
+For over 35 years, Primeur have been helping companies from all over the world to accelerate their data integration projects, ensuring their customers reach their business goals while making the most of their existing IT investments and avoiding unnecessary expenses. To help their clients achieve this, Primeur established their data integration platform, link:https://www.primeur.com/data-one[DATA ONE], in 2020 which is designed to federate different technologies, platforms, data types and deployment models. Primeur embed the Open Liberty Java runtime in DATA ONE, using it to power the core data integration choreographies. 
+
+
+== Why choose Open Liberty?
+
+Primeur says they fell in love with Open Liberty when using it within this project. The two greatest features that Primeur values from Liberty are its invisibility and observability. Although these features may seem like a contradiction and might not be something you have considered when selecting a cloud-native runtime, these two factors are critical for Primeur’s business. This is because the perspective of an ISV embedding Open Liberty inside its products is different from that of a business customer installing Open Liberty (whether on-prem or in a cloud environment) to run its business application.
```

Comment:
```suggestion
Primeur says they fell in love with Open Liberty when using it within this project. The two greatest features that Primeur values from Liberty are its invisibility and observability. Although these features may seem contradictory and might not be something you have considered when selecting a cloud-native runtime, they are critical for Primeur’s business. This is because the perspective of an ISV embedding Open Liberty inside its products is different from that of a business customer installing Open Liberty to run its business application, whether on-prem or in a cloud environment.
```